semantic_conventions/resource/container.yaml

```
@@ -12,10 +12,15 @@ groups:
       - id: id
         type: string
         brief: >
-          Container id. Usually a UUID, as for example used to
+          Container ID. Usually a UUID, as for example used to
           [identify Docker containers](https://docs.docker.com/engine/reference/run/#container-identification).
           The UUID might be abbreviated.
         examples: ['a3bf90e006b2']
+      - id: runtime
+        type: string
+        brief: >
+          Container runtime running the container.
+        examples: ['docker', 'containerd', rkt']
```

```suggestion
        examples: ['docker', 'containerd', 'rkt']
```
Otherwise this is interpreted as `rkt'`.

---

Good catch, this was a typo.